openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -718,9 +721,10 @@ For cryptographic binding, the Client has the following options defined in (#cre
 A Client makes a Credential Request to the Credential Endpoint by sending the following parameters in the entity-body of an HTTP POST request using the `application/json` media type.
 
 * `format`: REQUIRED when the `credential_identifiers` parameter was not returned from the Token Response. It MUST NOT be used otherwise. It is a String that determines the format of the Credential to be issued, which may determine the type and any other information related to the Credential to be issued. Credential Format Profiles consist of the Credential format specific parameters that are defined in (#format-profiles). When this parameter is used, the `credential_identifier` Credential Request parameter MUST NOT be present.
-* `proof`: OPTIONAL. Object containing the proof of possession of the cryptographic key material the issued Credential would be bound to. The `proof` object is REQUIRED if the `proof_types_supported` parameter is non-empty and present in the `credential_configurations_supported` parameter of the Issuer metadata for the requested Credential. The `proof` object MUST contain the following:
-    * `proof_type`: REQUIRED. String denoting the key proof type. The value of this parameter determines other parameters in the key proof object and its respective processing rules. Key proof types defined in this specification can be found in (#proof-types).
 * `credential_identifier`: REQUIRED when `credential_identifiers` parameter was returned from the Token Response. It MUST NOT be used otherwise. It is a String that identifies a Credential that is being requested to be issued. When this parameter is used, the `format` parameter and any other Credential format specific parameters such as those defined in (#format-profiles) MUST NOT be present.
+* `proof`: OPTIONAL. Object providing a single proof of possession of the cryptographic key material to which the issued Credential instance SHALL be bound to. `proof` MUST NOT be present if `proofs` is used.  The `proof` object MUST contain the following:
```

Comment:
is `SHALL` really necessary here?

```suggestion
* `proof`: OPTIONAL. Object providing a single proof of possession of the cryptographic key material to which the issued Credential instance will be bound to. `proof` parameter MUST NOT be present if `proofs` parameter is used.  The `proof` object MUST contain the following:
```

Reply:
my thinking was that the ultimate decision what to issue sits with the issuer, so SHALL/MUST was too strong, but I also don't see why the issuer would not bind an issued credential to the wallet-provided key material, so `will be` was a good balance. cc @bc-pi , curious what you would think

Reply:
In general, I tend to prefer not using the uppercase keywords unless there's an obvious need/value in doing so. That's pretty subjective, of course, but `will` seems okay here.

Reply:
noting that `proofs` currently has similar language "issued Credential instances SHALL be bound to."

Reply:
ok, I will remove the SHALL from all occurances around proof(s)